ansible/playbook/tasks/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+    - name: Packages                   | Add Nginx repository
+      yum_repository:
+        name: nginx
+        description: nginx repo
+        baseurl: http://nginx.org/packages/centos/7/$basearch/
+        gpgcheck: no
+        enabled: yes
+
+    - name: Install nginx rpm          | Install nginx rpm
+      yum:
+        name:
+          - nginx-1.20.1
```

Comment:
Remember, we had a plan on preventing yum from upgrading nginx to the latest version?

Reply:
Also, are these redundant now? https://github.com/percona/pmm-update/pull/134/files#diff-0180ac272b00bc7b2bb15d3b9b054cb60171c3a7022f057caa0136edad5295d3L127-L128

Reply:
We planned but I'm afraid we can't. We actually don't use the nginx repo at all. We're getting the nginx packages from `epel` repo and I decided just disable the nginx repo.